app/models/rdv.rb

```diff
@@ -76,6 +76,13 @@ def send_reminder
     end
   end
 
+  def send_cancellation_notifications
```

Comment:
I would extract this method to a service, I don't think it belongs in the model, it's very transactional

Reply:
also, because both the user and the agent can cancel (I think), I would rename this to be something like `SendRdvCanceledByAgentNotificationsService`.

or actually, I think we could do the actual update in the service and rename it `CancelRdvByAgentService`